### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by Draxin** [farag2/Sophia-Script-for-Windows#573] [Q&A] [answered]
I have this problem, I am using powershell 7.4 x64 and windows defender disabled
![Captura de pantalla 2024-06-10 051345](https://github.com/farag2/Sophia-Script-for-Windows/assets/60172166/ec059f53-530b-44e8-b055-458f4608de67)

**Comment by farag2** [comment, marked as answer]
What's the problem to reboot your PC? I do not understand.

**Comment by Draxin** [comment]
The script does not execute any function, only that appears

**Comment by farag2** [comment]
Do what it told if you can read.
